command/src/main/java/com/iluwatar/command/Wizard.java

```diff
@@ -40,7 +40,9 @@
   private Deque<Command> undoStack = new LinkedList<>();
   private Deque<Command> redoStack = new LinkedList<>();
 
-  public Wizard() {}
+  public Wizard() {
```

Comment:
Is there any use for keeping this constructor explicitly?

Reply:
Not really but I wanted to keep the change as small as possible.

Reply:
Yes, we should probably remove it.